Open Judge System/LocalWorker/OJS.LocalWorker/OjsSubmissionProcessingStrategy.cs

```diff
@@ -245,9 +250,10 @@ private void SaveParticipantScore()
                     {
                         this.participantScoresData.AddBySubmissionByUsernameAndIsOfficial(
                             this.submission,
-                            participant.UserName,
+                            participant.User.UserName,
```

Comment:
Why not directly participant.UserName?

Reply:
The participant has no userName, before it had because there is Select which takes IsOfficial and UserName.

Reply:
This will make extra query to get the User. Eager loading is the better approach here.